### DISCUSSION TITLE
Install specific version of SO on forward nodes?

### DISCUSSION THREAD

**Discussion by RollingMoses** [Security-Onion-Solutions/securityonion#4740] [Unsupported Versions]
In a distributed deployment, if I install a new forward node, does it automatically try to pull down and install the latest version of SO, or match what's on the manager? Is there a way to specify the version, just in case? I'm sitting at version 2.3.50 on the grid and would like to keep it there for now.

**Comment by weslambert** [comment]
The forward node will check to see what version of the software the manager has, then pull any changes as needed.  You could install from an older ISO, but otherwise, we do not provide an option to explicitly install a specific version.
